test/integration/server/inspec/server_spec.rb

```diff
@@ -36,32 +32,46 @@
 
 describe file('/root/.my.cnf') do
   it { should be_file }
-  its(:content) { should match(/jzYY0cQUnPAMcqvIxYaC/) }
+  its('content') { should match(/jzYY0cQUnPAMcqvIxYaC/) }
 end
 
 describe command('sysctl vm.swappiness') do
-  its(:stdout) { should match(/vm.swappiness = 0/) }
-  its(:exit_status) { should eq 0 }
+  its('stdout') { should match(/vm.swappiness = 0/) }
+  its('exit_status') { should eq 0 }
 end
 
 describe file('/etc/sysctl.d/99-chef-vm.swappiness.conf') do
-  its(:content) { should match(/vm.swappiness = 0/) }
+  its('content') { should match(/vm.swappiness = 0/) }
 end
 
 describe yumrepo('percona-noarch') do
```

Comment:
``[DEPRECATION] `yumrepo(reponame)` is deprecated.  Please use `yum.repo(reponame)` instead.``